tests/tests_E0_evaluation.py

```diff
@@ -0,0 +1,76 @@
+import pickle
+import os
+import shutil
+import logging
+import mock
+
+import src.A0_initialization as initializing
+import src.B0_data_input_json as data_input
+import src.C0_data_processing as data_processing
+import src.D0_modelling_and_optimization as modelling
+import src.E0_evaluation as evaluation
+
+from .constants import TEST_REPO_PATH, INPUT_FOLDER
+
+PARSER = initializing.create_parser()
+TEST_INPUT_PATH = os.path.join(TEST_REPO_PATH, INPUT_FOLDER)
+TEST_OUTPUT_PATH = os.path.join(TEST_REPO_PATH, "MVS_outputs")
+
+DICT_BEFORE = os.path.join(TEST_REPO_PATH, "dict_values_before_E0.pickle")
+
+DICT_AFTER = os.path.join(TEST_REPO_PATH, "dict_values_after_E0.pickle")
+
+
+@mock.patch(
+    "argparse.ArgumentParser.parse_args",
+    return_value=PARSER.parse_args(["-i", TEST_INPUT_PATH, "-o", TEST_OUTPUT_PATH]),
+)
+def setup_module(m_args):
+    """Run the simulation up to module E0 and save dict_values before and after evaluation"""
+    if os.path.exists(TEST_OUTPUT_PATH):
+        shutil.rmtree(TEST_OUTPUT_PATH, ignore_errors=True)
+    user_input = initializing.process_user_arguments()
+
+    logging.debug("Accessing script: B0_data_input_json")
+    dict_values = data_input.load_json(
+        user_input["path_input_file"],
+        path_input_folder=user_input["path_input_folder"],
+        path_output_folder=user_input["path_output_folder"],
+        move_copy=False,
+    )
+    logging.debug("Accessing script: C0_data_processing")
+    data_processing.all(dict_values)
```

Comment:
I figured that E0 will always be used after A-D functions have produced a `dict_values` variable. I did not want to use a static file, so that the test fails if the fields required at input of E0 if anything changed in A-D functions.

Reply:
Okay. Will have to see if I should myself adapt my tests then - they are almost all independent of the actual dict_values generated from A-D.